.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: weekly
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: weekly
```

Comment:
i don't think we want the dependabot updates for gomod.
for github-actions, maybe.

i think we can remove the dependabot.yml for now and add it in the future.

Reply:
Agree.

@ErikJiang  You may add one in your fork repo and check if the dependency update PR is needed.

Reply:
Got it. I'm done. 😀